server/svix-server/src/core/types.rs

```diff
@@ -463,15 +463,11 @@ impl<'de> Deserialize<'de> for EndpointSecret {
 
 impl Validate for EndpointSecret {
     fn validate(&self) -> std::result::Result<(), ValidationErrors> {
-        let re = format!(
```

Comment:
I don't fully understand what's going on here, but it looks like you're changing the code itself, not just the test (so commit message is wrong), but also unclear why you're making the changes.

Reply:
Since the secret is converted to a `Vec<u8>` as part of deserialization, the regex validation of the key prefix was completely artificial.